.infrastructure/deployment.yml

```diff
@@ -45,8 +45,27 @@ spec:
         - name: SECRET_KEY
           valueFrom:
             secretKeyRef:
-              name: app-secret
+              name: mysql-secrets
```

Comment:
The secret name `mysql-secrets` used here does not match the secret name `app-secret` defined in the volumes section. Ensure consistency in secret names to avoid runtime errors.